conf/messages-ecc.cy

```diff
@@ -310,55 +278,3 @@ ecc.address-lookup.postcode.no-results.manual = Hoffwn nodi fy nghyfeiriad â ll
-
-ecc.business-short-name-yes-no.company.title = A yw’ch cwmni’n defnyddio enw byrrach?
-ecc.business-short-name-yes-no.partnership.title = A yw’ch partneriaeth yn defnyddio enw byrrach?
-ecc.business-short-name-yes-no.charity.title = A yw’ch elusen yn defnyddio enw byrrach?
-ecc.business-short-name-yes-no.organisation.title = A yw’ch sefydliad yn defnyddio enw byrrach?
-ecc.business-short-name-yes-no.hint = Er enghraifft, mae Cyllid a Thollau Ei Mawrhydi hefyd yn cael ei alw’n CThEM.
-ecc.business-short-name-yes-no.company.empty = Rhowch wybod i ni a yw’ch cwmni’n defnyddio enw byrrach
-ecc.business-short-name-yes-no.partnership.empty = Rhowch wybod i ni a yw eich partneriaeth yn defnyddio enw byrrach
-ecc.business-short-name-yes-no.organisation.empty = Rhowch wybod i ni a yw’ch sefydliad yn defnyddio enw byrrach
-ecc.business-short-name-yes-no.charity.empty = Rhowch wybod i ni a yw’ch elusen yn defnyddio enw byrrach
-
-ecc.business-short-name.company.title = Beth yw enw byr eich cwmni?
-ecc.business-short-name.partnership.title = Beth yw enw byr eich partneriaeth?
-ecc.business-short-name.charity.title = Beth yw enw byr eich elusen?
-ecc.business-short-name.organisation.title = Beth yw enw byr eich sefydliad?
-
-ecc.business-short-name.short-name = Enw byr
-
-ecc.business-short-name.company.empty = Nodwch enw byrrach eich cwmni
-ecc.business-short-name.partnership.empty = Nodwch enw byrrach eich partneriaeth
-ecc.business-short-name.charity.empty = Nodwch enw byrrach eich elusen
-ecc.business-short-name.organisation.empty = Nodwch enw byrrach eich sefydliad
-ecc.business-short-name.error = Rhaid i’r enw byrrach fod yn 30 o gymeriadau neu lai
-
-ecc.subscription.outcomes.success.xi=Os ydych yn symud nwyddau i Ogledd Iwerddon neu oddi yno
-ecc.subscription.outcomes.success.xi.date.info=bydd angen rhif EORI sy’n dechrau gyda XI arnoch os ydych yn:
-ecc.subscription.outcomes.success.xi.info.bullet1=symud nwyddau rhwng Gogledd Iwerddon a gwledydd nad ydynt yn rhan o’r UE
-ecc.subscription.outcomes.success.xi.info.bullet2=gwneud datganiad yng Ngogledd Iwerddon
-ecc.subscription.outcomes.success.xi.info.bullet3=cael penderfyniad tollau yng Ngogledd Iwerddon
-ecc.subscription.outcomes.success.xi.info.href=<a href="/customs-enrolment-services/register/xi-eori-guidance" target="_blank" rel="noopener noreferrer">Gwnewch gais am rif EORI sy’n dechrau gyda XI.</a>
-ecc.subscription.outcomes.success.xi.info2=Os oes gennych rif EORI a gyhoeddwyd gan wlad yn yr UE, nid oes angen i chi gael EORI sy’n dechrau gyda XI.
-
-ecc.subscription.status.gg.h1=Eich rhif EORI a’ch cyfrif Porth y Llywodraeth
-ecc.subscription.status.gg.p1=TRANSLATE Your EORI number is linked to the Government Gateway account you have used for this application and you will have access to {0} within two hours.
-ecc.subscription.status.gg.p2=Ni allwch wneud cais am rif EORI arall gan ddefnyddio’r cyfrif Porth y Llywodraeth hwn.
```

Comment:
is this used in registration only ?

Reply:
Yes, I didn't find any usage of it in the subscription journey. It was used on subscription_outcome page where we had XI eori paragraph.

The message key is misleading because it's pointing to subscription.